src/ninja.cc

```diff
@@ -1025,7 +1012,7 @@ int ExceptionFilter(unsigned int code, struct _EXCEPTION_POINTERS *ep) {
 /// Returns an exit code, or -1 if Ninja should continue.
 int ReadFlags(int* argc, char*** argv,
               Options* options, BuildConfig* config) {
-  config->parallelism = GuessParallelism();
+  config->parallelism = GetProcessorCount();
```

Comment:
Does this need to handle 0 like the previous code?

Reply:
If `GetProcessorCount()` can return 0, then we should fix that, not the callers.